### ISSUE TITLE
Error Videoplayer.java

### ISSUE THREAD

**Issue by Anneleen** [macdonst/VideoPlayer#9]
Hi,

I tried implementing this plugin like you explained in your readme file, however I'm getting an error in the videoplayer.java file. 

On this line: FileOutputStream out = this.cordova.getActivity().openFileOutput(fileTo, Context.MODE_WORLD_READABLE);

it says 'Using MODE_WORLD_READABLE when creating files can be risky, review carefully'. 

What can I do about it?

**Comment by macdonst** [comment]
Yeah, MODE_WORLD_READABLE was just deprecated in Android 4.2. You are safe to continue using it for now as it won't disappear from the Android SDK for awhile yet. It's not really a security risk since what we are doing here is copying the video file from the assets folder to the internal storage application directory. In order for other programs like the video player intent to be able to read that file is has to be MODE_WORLD_READABLE. 

I know what needs to be done to fix it. It's detailed in this SO answer by Mark Murphy I just haven't had time to get to it yet.

http://stackoverflow.com/a/13860467/41679

Thanks for opening the issue. This way I won't forget to come back to it.
